--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -124,7 +124,7 @@ find_package(Belr 5.3.0 REQUIRED)
 if(ENABLE_ADVANCED_IM)
 	bc_find_package(XercesC XercesC::XercesC xerces-c REQUIRED)
 	set(HAVE_ADVANCED_IM 1)
-	set(LIBXSD_INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/libxsd")
+	set(LIBXSD_INCLUDE_DIRS "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
 endif()
 
 if(ENABLE_SRTP)
--- a/src/xml/linphone-imdn.h
+++ b/src/xml/linphone-imdn.h
@@ -59,7 +59,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/conference-info.h
+++ b/src/xml/conference-info.h
@@ -58,7 +58,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/imdn.h
+++ b/src/xml/imdn.h
@@ -59,7 +59,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/is-composing.h
+++ b/src/xml/is-composing.h
@@ -58,7 +58,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/rlmi.h
+++ b/src/xml/rlmi.h
@@ -58,7 +58,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/xml.h
+++ b/src/xml/xml.h
@@ -58,7 +58,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
--- a/src/xml/resource-lists.h
+++ b/src/xml/resource-lists.h
@@ -59,7 +59,7 @@
 
 #include <xsd/cxx/config.hxx>
 
-#if (XSD_INT_VERSION != 4000000L)
+#if (XSD_INT_VERSION < 4000000L)
 #error XSD runtime version mismatch
 #endif
 
