diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 99d48b33ef214ee5aed191f589266f1b0bccadfa..61cd637c742a9771bcf19927ac45b546f2d3e179 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -32,7 +32,7 @@ set(PUBLIC_LINK_LIBS
 		PRIVATE ${LINK_LIBS}
 	)
 	if (ENABLE_FLEXIAPI)
-		target_link_libraries(linphone-static PUBLIC jsoncpp_static)
+		target_link_libraries(linphone-static PUBLIC jsoncpp_lib)
 	endif()
 
 

