--- a/coreapi/linphonecore.c
+++ b/coreapi/linphonecore.c
@@ -1721,7 +1721,7 @@ static int _linphone_core_tls_postcheck_callback(void *data, const bctbx_x509_ce
 }
 
 static void certificates_config_read(LinphoneCore *lc) {
-	string rootCaPath = static_cast<PlatformHelpers *>(lc->platform_helper)->getDataResource("rootca.pem");
+	string rootCaPath = "/etc/pki/tls/cert.pem";
 	const char *rootca = linphone_config_get_string(lc->config, "sip", "root_ca", nullptr);
 
 	// If rootca is not existing anymore, we try data_resources_dir/rootca.pem else default from belle-sip
diff --git a/wrappers/java/classes/org/linphone/core/tools/AndroidPlatformHelper.java b/wrappers/java/classes/org/linphone/core/tools/AndroidPlatformHelper.java
index 8a91880..c515a29 100644
--- a/wrappers/java/classes/org/linphone/core/tools/AndroidPlatformHelper.java
+++ b/wrappers/java/classes/org/linphone/core/tools/AndroidPlatformHelper.java
@@ -340,7 +340,7 @@ public class AndroidPlatformHelper {
         /*legacy code for 3.X*/
         String basePath = mContext.getFilesDir().getAbsolutePath();
         //make sure to follow same path as unix version of the sdk
-        String mLinphoneRootCaFile = basePath + "/share/linphone/rootca.pem";
+        String mLinphoneRootCaFile = "/etc/pki/tls/cert.pem";
         String mRingSoundFile = basePath + "/share/sounds/linphone/rings/notes_of_the_optimistic.mkv";
         String mRingbackSoundFile = basePath + "/share/sounds/linphone/ringback.wav";
         String mPauseSoundFile = basePath + "/share/sounds/linphone/rings/dont_wait_too_long.mkv";
