diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2018-01-01 16:02:01.814039622 +0200
+++ ./CMakeLists.txt	2018-01-01 16:06:08.532813253 +0200
@@ -363,7 +363,7 @@ configure_file(cmake/LinphoneConfig.cmak
 	@ONLY
 )
 
-set(CONFIG_PACKAGE_LOCATION "${CMAKE_INSTALL_DATADIR}/Linphone/cmake")
+set(CONFIG_PACKAGE_LOCATION "${CMAKE_INSTALL_DATADIR}/Linphone/cmake" CACHE PATH "CMake configuration file location")
 install(EXPORT ${EXPORT_TARGETS_NAME}Targets
 	FILE LinphoneTargets.cmake
 	DESTINATION ${CONFIG_PACKAGE_LOCATION}
diff -up ./wrappers/cpp/CMakeLists.txt.orig ./wrappers/cpp/CMakeLists.txt
--- ./wrappers/cpp/CMakeLists.txt.orig	2018-01-01 16:16:35.197367914 +0200
+++ ./wrappers/cpp/CMakeLists.txt	2018-01-01 16:23:42.284381835 +0200
@@ -78,10 +78,10 @@ configure_file(LinphoneCxxConfig.cmake.i
 
 install(EXPORT LinphoneCxxTargets
 	FILE LinphoneCxxTargets.cmake
-	DESTINATION "${CMAKE_INSTALL_DATADIR}/LinphoneCxx/cmake"
+	DESTINATION "${CONFIG_PACKAGE_LOCATION}/../LinphoneCxx"
 )
 install(FILES
 	"${CMAKE_CURRENT_BINARY_DIR}/LinphoneCxxConfig.cmake"
 	"${CMAKE_CURRENT_BINARY_DIR}/LinphoneCxxVersion.cmake"
-	DESTINATION "${CMAKE_INSTALL_DATADIR}/LinphoneCxx/cmake"
+	DESTINATION "${CONFIG_PACKAGE_LOCATION}/../LinphoneCxx"
 )
